src/timeline/NowIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {View, TextStyle, ViewStyle} from 'react-native';
+import {HOUR_BLOCK_HEIGHT} from './Packer';
+
+export {HOUR_BLOCK_HEIGHT} from './Packer';
+
+export interface NowIndicatorProps {
+  styles: {[key: string]: ViewStyle | TextStyle};
+}
+
+const TodayIndicator = (props: NowIndicatorProps) => {
```

Comment:
Maybe better to call it `NowIndicator` here as well

Reply:
Right, missed that one. Fixed!